.github/CONTRIBUTING.md

```diff
@@ -67,12 +67,12 @@ In case you are suggesting a new feature, we will match your idea with our curre
 
     ```bash
     #npm
-    npm install
+    npm run setup
     npm link
     npm link webpack-cli
     ---------------------------
     #yarn
-    yarn
+    yarn run setup
```

Comment:
I think strictly enforcing new contributors to install dependencies for side packages isn't a good idea as PR may not be related to the sub packages and just adds extra friction.
If user wants to, they should follow the steps at "Bootstrap all the submodules before building for the first time" heading below.